exec/java-exec/src/main/java/org/apache/drill/exec/ops/ContextInformation.java

```diff
@@ -28,12 +28,14 @@
   private final String currentDefaultSchema;
   private final long queryStartTime;
   private final int rootFragmentTimeZone;
+  private final int hllMemoryLimit;
 
   public ContextInformation(final UserCredentials userCredentials, final QueryContextInformation queryContextInfo) {
     this.queryUser = userCredentials.getUserName();
     this.currentDefaultSchema = queryContextInfo.getDefaultSchemaName();
     this.queryStartTime = queryContextInfo.getQueryStartTime();
     this.rootFragmentTimeZone = queryContextInfo.getTimeZone();
+    this.hllMemoryLimit = queryContextInfo.getHllMemoryLimit();
```

Comment:
This is labeled as a memory limit here, but the description of the option speaks of bit counts. This is confusing. A memory limit is usually expressed in terms of byte (or MB) that some operator may consume. A bit count might give rise to a memory usage, but is not really a limit per-se...